lib/rpm_package.py

```diff
@@ -205,6 +205,11 @@ def _load(self):
                     dep_name, self.distro, force_rebuild=self.force_rebuild)
                 self.build_dependencies.append(dep)
 
+            # keeps backward compatibility with old yaml files which have build dependencies
```

Comment:
Several lines in our code (including this file) have more than 80 columns and we are not reviewing this exact line limit in any of our code reviews. Pls let us be consistent in code review. This is as minor as it gets. I'll alter it here anyway.